Library/Formula/sgfutils.rb

```diff
@@ -0,0 +1,68 @@
+class Sgfutils < Formula
+  desc "collection of command-line utilities to work with SGF files"
+  homepage "http://homepages.cwi.nl/~aeb/go/sgfutils/html/sgfutils.html"
+  url "http://homepages.cwi.nl/~aeb/go/sgfutils/sgfutils-0.22.tgz"
+  sha256 "4ad83be91ac3a52dd8bd0766bfe403499430da96e680c5aea2f5381ba0b34923"
+
+  depends_on "openssl" => :build
+
+  # Makefile tweaks for OS X. The corresponding patch was sent to the
+  # main author of sgfutils on 2016-03-17 for potential upstream
+  # inclusion.
```

Comment:
The patch will not be accepted upstream since it breaks the build on GNU systems. GNU libc does not have an actual `libiconv.so`; everything is just built into libc directly.

However, when re-reading the relevant section of the GNU make documentation, I found out that it should be enough to pass `LDFLAGS=-liconv` to make, so patching the upstream Makefile is not necessary after all.

I'll push an amended commit once `brew audit` is done.